src/renderer/components/message/MessageMarkdown.tsx

```diff
@@ -111,23 +111,19 @@ function renderElementPreview(elm: ParsedElement, key?: number): JSX.Element {
       )
 
     case 'StrikeThrough':
-      return <s key={key}>{elm.c.map(renderElement)}</s>
+      return <s key={key}>{elm.c.map(renderElementPreview)}</s>
 
     case 'Italics':
-      return <i key={key}>{elm.c.map(renderElement)}</i>
+      return <i key={key}>{elm.c.map(renderElementPreview)}</i>
 
     case 'Bold':
-      return <b key={key}>{elm.c.map(renderElement)}</b>
+      return <b key={key}>{elm.c.map(renderElementPreview)}</b>
```

Comment:
This fixes a separate issue, doesn't it? It's worth mentioning it in the commit and the changelog I think if so.